global.json

```diff
@@ -1,4 +1,8 @@
 {
+  "sdk": {
```

Comment:
This is fine, but lets not have the version in two places.  Remove the version from the pipeline YAML and update the UseDotNet task like this:

```diff
variables:
  ArtifactsDirectoryName: 'artifacts'
  BinaryLoggerArgs: '"/BinaryLogger:$(Build.SourcesDirectory)\$(ArtifactsDirectoryName)\msbuild.binlog"'
  BuildConfiguration: 'Debug'
  BuildPlatform: 'Any CPU'
-  DotNetCoreVersion: '3.1.300'
...
  - task: UseDotNet@2
-    displayName: 'Install .NET Core $(DotNetCoreVersion)'
+    displayName: 'Install .NET Core from global.json'
    inputs:
-      version: '$(DotNetCoreVersion)'
+      useGlobalJson: true
```

Reply:
I didn't know this works. Hell yeah. Give me a sec.